apps/style/common.scss

```diff
@@ -1120,6 +1120,14 @@ $workspace-header-button-margin: ($workspace-headers-height - $workspace-header-
   tr:nth-child(even){
     background-color: $lightest_purple;
   }
+
+  #template-level-warning {
```

Comment:
I'm curious why you chose to put this in `common.css` instead of making it an inline style

Reply:
It looked like that's where the rest of the styling or this dialog lived (e.g. for the shaded lines)?  I thought about making it an inline style but that seemed like it was just introducing another pattern into this file.  I'm hoping Tim will take a look with LP-85 and make some more dialog-wide changes if they decide it's the best way forward.

Reply:
I just want to point out that there are several instances of inline styles in the file for example, https://github.com/code-dot-org/code-dot-org/blob/94af9bdcf767d9bbd60379e64bc41255bbaae494/apps/src/templates/VersionHistory.jsx#L193. I'd say since this style is specific to the VersionHistory component and isn't shared elsewhere, it might be worth making it an inline style since that seems to be the pattern we generally use.

Reply:
Yeah, I noticed this too.  The pattern is...  unclear.  😬